src/app/tracing/configuration/filter-table-view/filter-table-view.component.ts

```diff
@@ -256,27 +256,49 @@ export class FilterTableViewComponent implements OnChanges, DoCheck, OnInit, OnD
         this.selectColumns.emit();
     }
 
-    onRowSelectionChange({ selected }: { selected: TableRow[] }): void {
-        if (this.processedInput__) {
-            const selectedRowIds = selected.map(row => row.id);
-            // dblclick events trigger 3 selection change events
-            // only the first one changes (usually) the selection
-            // we check here the selection change to emit only true selection changes
-            if (this.areSelectedRowIdsEqual(selectedRowIds, this.processedInput__.selectedRowIds)) {
-                // no selection change
-            } else {
-                // selection change detected
-                this.processedInput__.selectedRowIds = selected.map(row => row.id);
+    isTableRowArray(toInspect: TableRow[] | { selected: TableRow[] } | any): toInspect is TableRow[] {
+        return (toInspect as TableRow[]).length !== undefined;
```

Comment:
Recommendation:
- Isn't it better to use `Array.isArray(toInspect)` instead of  `(toInspect as TableRow[]).length !== undefined`?